src/containers/Onboarding/CreateGroup.js

```diff
@@ -40,7 +40,17 @@ class CreateGroup extends Component {
   * @param {Object} fields
   */
   _handleSubmit(fields) {
-    return this.props.createGroup(fields);
+    const promise = new Promise((resolve) => {
+      resolve(this.props.createGroup(fields));
+    });
+
+    promise.then((val) => {
+      // this.props.createOrderFrequency(val);
```

Comment:
Yep

https://github.com/coopdevs/katuma-web/blob/develop/src/redux/modules/orders_frequencies/orders_frequencies.js#L171

Just add `import { create } from 'redux/modules/orders_frequencies/orders_frequencies';` at `#12` in this file and then pass it to the component through the `connect()`, as we do for `createGroup()` at the bottom of this file.